### PR TITLE
Change to uPort's keypair account type

### PR DESCRIPTION
**Pull request by tinybike**
[Clubhouse Story](https://app.clubhouse.io/augur/story/9345)

## Submitter checklist
- [x] Test covering functionality added/fixed in
- Check the linked story includes the following:
  - [ ] Steps to reproduce
  - [ ] Screenshot (If applicable)

## Reviewer Checklist
- [ ] Test/lint pass

### CODE REVIEW

File: src/modules/auth/actions/login-with-uport.js [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
-    dispatch(loadAccountData({
-      ...account,
-      meta: {
-        address: account.address,
-        signer: txObj => uPortSigner(txObj, dispatch),
-        accountType: augur.rpc.constants.ACCOUNT_TYPES.U_PORT,
-      },
-    }))
-  }
+export const loginWithUport = account => (dispatch) => {
```
Comment (adrake33) [inline src/modules/auth/actions/login-with-uport.js]: Why was this changed from being an `export default`? I thought we were trying to use `export default` whenever possible.

Comment (tinybike) [reply src/modules/auth/actions/login-with-uport.js]: Just for consistency, generally export const is used else where.
